cirq-core/cirq/protocols/measurement_key_protocol_test.py

```diff
@@ -19,50 +19,53 @@
 
 def test_measurement_key():
     class ReturnsStr:
-        def _measurement_key_(self):
+        def _measurement_key_str_(self):
             return 'door locker'
 
     assert cirq.is_measurement(ReturnsStr())
-    assert cirq.measurement_key(ReturnsStr()) == 'door locker'
+    with cirq.testing.assert_deprecated(deadline="v0.13"):
```

Comment:
Please add a similar test for a class which defines `_measurement_key_`.

Reply:
Done